math/src/stableswap/math.rs

```diff
@@ -221,11 +386,11 @@ pub(crate) fn calculate_y_given_in<const N: u8, const N_Y: u8>(
 		.map(|(idx, v)| if idx == idx_in { new_reserve_in } else { *v })
 		.collect();
 
-	calculate_y::<N_Y>(&xp, d, amplification)
+	calculate_y_internal::<Y>(&xp, d, amplification)
 }
 
-/// Calculate new amount of reserve IN given amount to be withdrawn from the pool
-pub(crate) fn calculate_y_given_out<const N: u8, const N_Y: u8>(
+/// Calculate new amount of reserve ID given amount to be withdrawn from the pool
```

Comment:
```suggestion
/// Calculate new amount of reserve IN given amount to be withdrawn from the pool
```
probably faulty auto-replace, right?